en-US/Players_Guide/03_Creating_a_Character/Classes/Classes.md

```diff
@@ -13,26 +13,26 @@
+|:------------|:-------------------------------------------------------|:-------------|:--------|:-----------------------|:------------------------------------------------------|:----------------------------|
+| Adept       | Martial Arts Specialist                                | None         | d8      | Dexterity, Wisdom      | None                                                  | Simple weapons, shortswords |
+| Bard        | Support caster                                         | Full         | d8      | Charisma               | Light armor                                           | Simple weapons, hand crossbows, longswords, rapiers, shortswords |
+| Cleric      | Healer & frontline protector                           | Full         | d8      | Wisdom                 | Light & medium armor, shields                         | Simple weapons              |
+| Dreadnought | Rage-fueled warrior                                    | None         | d12     | Strength, Constitution | Light & medium armor, shields                         | Simple & martial weapons    |
+| Fighter     | Versatile weapon specialist                            | None         | d10     | Strength or Dexterity  | All armor, shields                                    | Simple & martial weapons    |
+| Ranger      | Hunter & tracker with nature magic                     | Half         | d10     | Dexterity, Wisdom      | Light & medium armor, shields                         | Simple & martial weapons    |
+| Rogue       | Stealthy skilled precision striker                     | None         | d8      | Dexterity              | Light armor                                           | Simple weapons, hand crossbows, longswords, rapiers, shortswords |
+| Sorcerer    | Arcane caster of flexible innate magic                 | Full         | d6      | Charisma               | None                                                  | Daggers, darts, slings, quarterstaffs, light crossbows |
+| Vanguard    | Principled warrior with some divine magic              | Half         | d10     | Strength, Charisma     | All armor, shields                                    | Simple & martial weapons    |
+| Warlock     | Caster drawing power from a pact with a powerful being | Full         | d8      | Charisma               | Light armor                                           | Simple weapons |
+| Wodewose    | Shapeshifting nature caster                            | Full         | d8      | Wisdom                 | Light & medium armor (non-metal), shields (non-metal) | Simple weapons (non-metal)  |
+| Wizard      | Arcane scholar with wide spell access                  | Full         | d6      | Intelligence           | None                                                  | Daggers, darts, slings, quarterstaffs, light crossbows |
```

Comment:
This wasn't strictly necessary, but I wanted the table to be more in line with the other tables we already have.